.github/workflows/get-secretmanager-secrets-it.yml

```diff
@@ -0,0 +1,47 @@
+name: get-secretmanager-secrets Integration
+on: [push]
+jobs:
+  gcloud:
+    name: with setup-gcloud
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: ./setup-gcloud
+      with:
+        service_account_email: ${{ secrets.SETUP_GCLOUD_IT_EMAIL }}
+        service_account_key: ${{ secrets.SETUP_GCLOUD_IT_KEY_B64 }}
```

Comment:
In order to properly test this, we need an envvar that has a b64-encoded service account json and a non-encoded one.